plugins/rtmp-services/data/services.json

```diff
@@ -2515,6 +2515,72 @@
                 "max video bitrate": 7200,
                 "max audio bitrate": 196
             }
+        },
+        {
+            "name": "Shareplay.tv",            
+            "stream_key_link": "https://shareplay.tv/no-stream",
```

Comment:
Is this the correct URL whereby users can retrieve their stream key? This doesn't look correct to me at first glance.

Reply:
I have updated this link. User login on our site and request for the steam key. we have shown the Get Steam key menu for login user

Reply:
@dodgepong ,
I have updated  services.json file.

Reply:
It looks like you completely removed this link from your additions. Was that intentional?

Reply:
We have at times one user live stream on Shareplay.  Any user lives stream on shareplay his/her First login and get stream key.  Remove URL is our live stream page URL. So I have removed it.